frontend/src/app/api/user/organizations/[organizationId]/users/[userId]/handler.ts

```diff
@@ -0,0 +1,59 @@
+import { readError, UnauthorizedError } from "src/errors";
+import { getSession } from "src/services/auth/session";
+import {
+  deleteOrganizationUser,
+  updateOrganizationUserRoles,
+} from "src/services/fetch/fetchers/organizationsFetcher";
+
+import { NextRequest, NextResponse } from "next/server";
+
+export const deleteOrganizationUserHandler = async (
+  _request: NextRequest,
+  options: { params: Promise<{ organizationId: string; userId: string }> },
+) => {
+  const session = await getSession();
+  if (!session?.token) {
+    throw new UnauthorizedError("Not authenticated");
+  }
+
+  const { organizationId, userId } = await options.params;
+
+  try {
+    const deletedUser = await deleteOrganizationUser(
+      session.token,
+      organizationId,
+      userId,
+    );
+    return NextResponse.json({ data: deletedUser }, { status: 200 });
```

Comment:
is status 200 necessary here? What does the response do if you don't provide it?